packages/scales/index.d.ts

```diff
@@ -23,6 +23,7 @@ declare module '@nivo/scales' {
         type: 'time'
         format?: string
         precision?: 'millisecond' | 'second' | 'minute' | 'hour' | 'month' | 'year' | 'day'
+        useUTC: boolean
```

Comment:
Sorry for the late review :/ This property is optional, it should be `useUTC?: boolean`.